service/RTC/StandardVideoSettings.ts

```diff
@@ -74,6 +75,19 @@ export const STANDARD_CODEC_SETTINGS = {
     }
 };
 
+export const VIDEO_CODECS_BY_COMPLEXITY = {
```

Comment:
What happens if someone configures H.264 as their preferred codec for either of the 2?

Reply:
The reason I haven't included H.264 is because I am not able to configure it using the codec selection API. So we never attempt to switch to it as part of runtime adaptation. However, if the client is currently encoding in H.264 and the QualityController attempts correction, it will pick the first one from the list here and walk down the list when needed.